Soyeon/MyProfile/MyAttractivenessScoreViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MyAttractivenessScoreViewController.swift
+//  Soyeon
+//
+//  Created by junyeong-cho on 2021/09/21.
+//  Copyright © 2021 ludus. All rights reserved.
+//
+
+import UIKit
+
+final class MyAttractivenessScoreViewController: UIViewController {
+    
+    @IBOutlet private weak var userStackView: UIStackView!
+    @IBOutlet private weak var scoreLabel: UILabel!
+    @IBOutlet private weak var soyeonRatingView: SoyeonRatingView!
+    @IBOutlet private weak var rescoreButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupLayout()
+        configureView()
+    }
+    
+    private func setupLayout() {
+        setNavigationTitle("나의 매력지수")
+    }
+    
+    private func configureView() {
+        let currentScore: Float = 4.5
+        scoreLabel.text = "\(currentScore)점"
+        soyeonRatingView.configure(
+            with: SoyeonRatingViewModel(
+                currentScore: currentScore,
+                levels: [
+                    SoyeonRatingViewModel.Level(score: 1.0, title: "브론즈"),
+                    SoyeonRatingViewModel.Level(score: 2.0, title: "실버"),
+                    SoyeonRatingViewModel.Level(score: 3.0, title: "골드"),
+                    SoyeonRatingViewModel.Level(score: 4.0, title: "루비"),
+                    SoyeonRatingViewModel.Level(score: 5.0, title: "다이아")
+                ]
+            ))
+    }
+    
+    @IBAction private func rescoreButtonTapped(_ sender: UIButton) {
+        SoyeonBasicAlertView
+            .alert(message: "매력지수를 새로 측정하려면\n5HP가 소모됩니다.")?
+            .action(style: .basic("아이템 충전하기"), completion: { [weak self] _ in
```

Comment:
이건 그냥 의견인데요! 
`completion closure`를 함수로 정의해두면 어떨까요?
어떤 행동을 할건지 함수 이름으로 보면 가독성이 좋아질것 같아서요!

 
```suggestion
            .action(style: .basic("아이템 충전하기"), completion: recalculateMyScore)
      ...
      
      private func recalculateMyScore() {
          // 재수집 API 호출
          progressMyScore()
       }
       
       private func progressMyScore() {
           // progress 노출
       
           self.soyeonRatingView.configure(
               with: SoyeonRatingViewModel(
                        currentScore: 0.0,
                        levels: [ 
                        ... 
       }
```

Reply:
관련로직을 함수로 정의하도록 할게요~!